utils/file/file.go

```diff
@@ -83,15 +83,20 @@ func GetPluginOrder(taskInfo *mesos.TaskInfo) ([]string, error) {
 		plugins = append(plugins, plugin)
 	}
 
-	log.Println("Plugin Order : ", plugins)
+	pod.PluginOrder = plugins
+	log.Println("Plugin Order : ", pod.PluginOrder)
 	return plugins, nil
 }
 
 // Get downloaded file paths from uris
 func GetYAML(taskInfo *mesos.TaskInfo) []string {
 	log.Println("====================Get compose file from URI====================")
 	var files []string
+	if taskInfo == nil || taskInfo.Executor == nil || taskInfo.Executor.Command == nil {
+		return files
+	}
 	uris := taskInfo.Executor.Command.GetUris()
+	fmt.Println("file********:", uris)
```

Comment:
Should we use `log.Println` instead to be consistent? Or is it a debugging statement you meant to take out?

Reply:
yes, forgot to take this out, thx